sdk/python/feast/feature_server.py

```diff
@@ -18,7 +19,12 @@
 from feast.constants import DEFAULT_FEATURE_SERVER_REGISTRY_TTL
 from feast.data_source import PushMode
 from feast.errors import PushSourceNotFoundException
+from prometheus_client import Gauge, start_http_server
+import psutil
 
+# Define prometheus metrics
```

Comment:
Maybe it's not in scope for this but I would recommend adding latency metrics as well to the write and read endpoints.